app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -4,6 +4,9 @@ class OmniauthCallbacksController < Devise::OmniauthCallbacksController
   def google_oauth2
     if user.present?
       user.accept_invitation! if invitation_token
+      if request.env['omniauth.auth'].try(:[], 'info').try(:[], 'image')
```

Comment:
What do you think of using the safe navigation operator `&.` instead of `try`? 

[Here's a good explanation on this!](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)